app/src/main/kotlin/at/bitfire/davdroid/sync/adapter/SyncFrameworkIntegration.kt

```diff
@@ -122,6 +120,17 @@ class SyncFrameworkIntegration @Inject constructor(
         ContentResolver.cancelSync(syncRequest)
     }
 
+    /**
+     * Cancels all Sync Adapter Framework syncs (system wide) for given account and
+     * authority. If authority is null, all syncs for the given account
+     * regardless of authority are canceled.
+     *
+     * @param account The account for which syncs should be canceled.
+     * @param authority Null or the authority for which syncs should be canceled.
+     */
+    fun cancelSync(account: Account, authority: String?) =
+        ContentResolver.cancelSync(account, authority)
```

Comment:
We use it only in the migration and probably never again. Honestly I'd not add it to the `SyncFrameworkIntegration` class.